tools/jil/server/asset-server.js

```diff
@@ -536,7 +543,8 @@ class AssetServer extends BaseServer {
     this.unitTestDir = path.resolve(__dirname, '../../../tests/browser')
     this.addHandler(this.serviceRequest.bind(this))
     this.router = new Router(this, testConfig, output)
-    this.agentTransform = new AgentInjectorTransform(this.buildDir, this, this.router)
+    this.browserifyTransform = new BrowserifyTransform(testConfig)
```

Comment:
Just curious on this one, what was the purpose of keeping this in memory?

Reply:
It is a remnant of me trying to fix the test runner for this PR. Ill take a look at removing it.